pkg/dyninst/testprogs/testdata/probes/sample.yaml

```diff
@@ -0,0 +1,254 @@
+binary: sample
+probes:
+- id: test_single_byte
+  type: LOG_PROBE
+  where:
+    method_name: main.test_single_byte
```

Comment:
Didn't all these functions get renamed to camelCase? How does this pass?

Reply:
Needed to rebuild the binaries locally, I totally missed that. Fixing.